superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -2861,7 +2858,7 @@ msgstr "Escolha uma fonte e um alvo"
 
 #: superset/assets/javascripts/explore/stores/visTypes.js:797
 msgid "Chord Diagram"
-msgstr "Diagrama de acordes"
+msgstr "Diagrama de Acordes"
```

Comment:
Don't know, but maybe, "Diagrama de cordas"

Reply:
Yes, diagrama de cordas sounds good for this diagram. Acordes is more related to music.